src/Teams/Controllers/MarkParserController.cs

```diff
@@ -1,13 +1,22 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
 using Westwind.AspNetCore.Markdown;
 
 namespace Teams.Controllers
 {
     public class MarkParserController : Controller
     {
+        private readonly ILogger<MarkParserController> _logger;
+
+        public MarkParserController(ILogger<MarkParserController> logger)
+        {
+            _logger = logger;
+        }
+
         [HttpPost]
         public JsonResult GetMarkdownQuestion(string question)
         {
+            _logger.LogInformation("");
```

Comment:
Why you decided to do not log question string?

Reply:
In HTTP Post methods without binding model (like [FromBody]), parameters are automatically logged.
![getMarkdown](https://user-images.githubusercontent.com/72085557/114277169-c17a6b80-9a32-11eb-891b-46349cc4fa6b.JPG)

It is possible thanks to [${aspnet-request-form}](https://github.com/NLog/NLog/wiki/AspNetRequest-Form-Layout-Renderer) layout renderer.